config.rb.in

```diff
@@ -4,6 +4,9 @@ module BRICR
   MAX_DATAPOINTS = Float::INFINITY
   #MAX_DATAPOINTS = 2
 
+  # used in run_seed_buildingsyncs to limit the number of properties to keep after each pull from SEED
+  PROPERTY_COUNT = 2
```

Comment:
I'm totally being curious over here...is this the max property count or a default value but can be changed per user?

Reply:
The PROPERTY_COUNT is used in the `run_seed_buildingsyncs.rb` file.  When the seed client pulls down the [search_results](https://github.com/NREL/bricr/blob/ae5ad6fcdbb5a0d394833a3bb9991386c9b23bc0/bin/run_seed_buildingsyncs.rb#L52), it pulls down max of 10,000.  As it is iterating through each of the items, PROPERTY_COUNT is basically used to [splice the array](https://github.com/NREL/bricr/blob/f9037b5de9744f16da6d11c19037636752579e20/bin/run_seed_buildingsyncs.rb#L75-L84) such that it doesn't hold onto all the potential 'Not Started' records.  This was done to avoid the duplication of building simulations across different servers.